openapi.yaml

```diff
@@ -3879,6 +3892,7 @@ paths:
                     "203.0.113.1",
                     "192.0.1.0/24"
                   ],
+                  "type": "g6-standard-1",
                   "updates" = {
```

Comment:
Noticed an old typo here, should be `"updates": {`

Reply:
Fixed.